templates/data-providers/index.jsx

```diff
@@ -147,6 +141,13 @@ const DataProvidersSearchTemplate = React.memo(
           >
             Become data provider
           </Button>
+
+          <RepositoriesMap
```

Comment:
You will need to add some padding here.

Reply:
I added a new line above, you see? Why doesn't it work? 😂

Reply:
hmm I am not a fan of this empty paragraph just for the button. I wonder you added it there since you care too much about using proper html elemenets.

Reply:
I think it's correct if buttons are grouped in a 'group' element. According to the spec, there are 3 types of elements:
1. Sectioning
2. Grouping content
3. Text-level

I think they should be included in the full hierarchy. It's not wrong if you have a button straight in the section but it's not wrong in the paragraph either. I prefer paragraphs for implicit and controllable paragraph spacing.